### PR TITLE
fs-1078; Makefile cleanup and additional features

### PR DESCRIPTION
**Pull request by jakeschuurmans**
Some QoL stuff I wanted to do to prepare for the actual work that will come a bit later today. So separating it out into two PRs.

Additions to the makefile:
-Moving the port declarations to the top of the makefile
-Flag that allows your portfowarding to exist on your LAN instead of just your machine.
-Moving the port forward to a function (right under the port declarations) to make the LAN enabling feature a bit cleaner
-Make target to just forward all ports and run in background. I got quite annoyed having to have multiple terminals open just to port forward what i needed.
-Make target to kill all port forwarding processes
-Increased the help character max to 32, as it was no longer cleanly printing out the help menu

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+ALLOY_PORT=9091
+HSS_PORT=8000
+CRDB_PORT=26257
+CHAOS_DASH_PORT=2333
+JAEGER_DASH_PORT=16686
+MINIO_PORT=9000
+
+# Makefile function. call with $(call,<param1>,<param2>...)
+define forward-port
+		$(if $(filter-out 1,$(FOWARD_TO_LOCAL_NETWORK)),kubectl port-forward $(1) $(2):$(2),kubectl port-forward $(1) --address 0.0.0.0 $(2):$(2))
```
Comment (joelrebel) [inline Makefile:13]: I'd prefer if its a separate command to port-forward on all IPs, by default this should just port forward locally, so we're not exposing anything externally unexpectedly

**Review by joelrebel — CHANGES_REQUESTED** [review]
Nice work here, looks good overall - left a comment on the port forward.

File: Makefile (hunk 1 above)
Comment (jakeschuurmans) [reply Makefile:13]: so something like `make port-forward-conditionorc-api-lan`?

Comment (joelrebel) [reply Makefile:13]: yep, or `port-forward-all-lan`

**Review by joelrebel — APPROVED** [review]

**Pull request merged**